packages/sources/config/rollup.config.js

```diff
@@ -46,15 +46,25 @@ const plugins = [
         keep_fnames: true
     }),
     postcss({
-        extract: true
+        extract: true,
+        minimize: true
     }),
     json()
 ];
 
 export default rollupConfig(
-    external(externalDeps({ ...dependencies, ...peerDependencies })),
+    external(externalDeps(
+        { ...dependencies, ...peerDependencies, ...devDependencies },
+        [ '@patternfly', '@redhat-cloud-services',  '@data-driven-forms', 'lodash' ]
```

Comment:
does the `externalDeps` add wildcards? (`@patternfly/**`)

Reply:
it transform deps to ``${item}/**``

Reply:
I assume it keeps the original as well